packages/frontier-plugins/plugin-vue/coverage/clover.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
coverage directory should not be commited. there should be a config that generates this folder as text-coverage

Reply:
added a gitignore for these folders